scripts/projector.md

```diff
@@ -0,0 +1,38 @@
+
+## Projector script documentation
+
+The projector installation and initialization script (see [`projector.sh`](projector.sh)) allows different levels of customization:
+
+### Projector config name
+
+Specify the **desired name of Projector config** by defining the `projector_config_name` variable.
```

Comment:
what is a Projector config? What do the different options do and which one should one choose?

Reply:
It's projector runnable instance.
I did remove this setting and forced the script-generated projector config to be named "strigo"
Sadly projector is more a interactive oriented CLI, so not easy to setup in script. Not much more to setup anyway (port [fixed to 9999], hostname [automatically set to Strigo dynamic DNS], TLS [will add it in script in future], password [just added it])
I added a section explaining how to edit the configuration afterwards, or add others.